### PR TITLE
Equipment UI

### PR DESCRIPTION
**Pull request by Mir3605**
Added equipment bar and methods to add and remove items from it. It should be enough for now, although it can be developed in the future. I didn't create nice graphics for it, as it should be done after the resolution is set to full HD.

Edit: adding a screenshot
Screenshot_168

### CODE REVIEW

File: Assets/Utility/UI/Equipment/item.gd [hunk 1]
```diff
@@ -0,0 +1,16 @@
+extends Node2D
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	pass # Replace with function body.
+
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _process(delta):
```
Comment (ddziechciarz) [inline Assets/Utility/UI/Equipment/item.gd:9]: get rid of all unused _ready and _process methods in all the files

**Review by ddziechciarz — CHANGES_REQUESTED** [review]
There are empty _ready and _process methods, delete them to improve code readability

File: Assets/Utility/UI/Equipment/item.gd (hunk 1 above)
Comment (Mir3605) [reply Assets/Utility/UI/Equipment/item.gd]: They were removed

**Review by ddziechciarz — APPROVED** [review]
git

**Pull request merged**